### PR TITLE
Campus Wizard View

### PR DESCRIPTION
**Pull request by lucernae**
Hi, this is a minimum working addition of Campus Wizard form.
Issue #54 

# What's in this PR

- We uses django-formtools for wizard form (multi step)
- Hook ford3 urls to core urls
- Create basic Django form for Step 1: Campus Details #51 
- Changed photo_url to photo FileField, because we want to upload the file

# Not on this PR

- The style is ugly. I only worked on the DOM, not yet for CSS
- Submit redirect, because not sure where to redirect after editing this Campus
- Step 2 to 4, because I still have some questions, and changes in the model and widget is necessary.

CC @pierrealixt @Christiaanvdm

### CODE REVIEW

**Comment by pierrealixt** [comment]
@lucernae 
After submitting, you are redirected to the Campus page:
Screen Shot 2019-03-18 at 19 27 46

What about a selenium test?  ¯\_(ツ)_/¯

File: django_project/ford3/views/views.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
-from django.shortcuts import render # noqa
+# coding=utf-8
 
-# Create your views here.
+from django.conf import settings
+from formtools.wizard.views import CookieWizardView
+
+
+class CampusWizard(CookieWizardView):
```
Comment (dimasciput) [inline django_project/ford3/views/views.py:7]: It would be better if it was made into its own file.

**Comment by lucernae** [comment]
> What about a selenium test? ¯_(ツ)_/¯

@pierrealixt haha, nothing to test about at this stage. Since the form doesn't do anything.

File: django_project/ford3/views/views.py (hunk 1 above)
Comment (lucernae) [reply django_project/ford3/views/views.py:7]: I agree. But then again the file will only contain these (there's no other view for Campus).
So, I was thinking of changing the `views` python file to something more generic, but doesn't have one in mind for now.

**Comment by lucernae** [comment]
Guys, I will merge this and continue with the other step.

**Pull request merged**